cf_xarray/options.py

```diff
@@ -42,6 +39,11 @@ def __init__(self, **kwargs):
         self._apply_update(kwargs)
 
     def _apply_update(self, options_dict):
+        for k, v in options_dict.items():
+            if k == "custom_criteria":
+                OPTIONS["custom_criteria"] = always_iterable(
+                    OPTIONS["custom_criteria"], allowed=(tuple, list)
+                )
         OPTIONS.update(options_dict)
```

Comment:
this will overwrite the earlier value

Reply:
Ok hopefully I fixed that.

Reply:
hmm.. now we're mutating user input. Let's do a `copy.deepcopy(options_dict)` first.

Reply:
https://github.com/xarray-contrib/cf-xarray/pull/243/commits/092d427794947ca021425b4272a49200a966841a ?